lib/src/main/java/com/dezen/riccardo/smshandler/SMSMessage.java

```diff
@@ -97,4 +97,14 @@ public String toString(){
         MESSAGE_EMPTY,
         MESSAGE_VALID
     }
+
+    @Override
+    public boolean equals(Object obj){
```

Comment:
It's convetion to add these two if at the beginning:
"
if(obj == null) return false;
if(this == obj) return true;
"

Reply:
Completely forgot, thanks